hpobench/config.py

```diff
@@ -1,3 +1,21 @@
+"""
```

Comment:
Why do we have the changelog here?

Reply:
Good question. I will move it to the main changelog.